docs/topics/Gateway.md

```diff
@@ -833,6 +846,45 @@ The inner `d` key is a boolean that indicates whether the session may be resumab
 
 `APPLICATION_COMMAND_PERMISSIONS_UPDATE` event, sent when an application command's permissions are updated. The inner payload is an [application command permissions](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-permissions-object-guild-application-command-permissions-structure) object.
 
+### Auto Moderation
+
+All auto moderation related events are currently only sent to bot users which have the `MANAGE_GUILD` permission.
+
+#### Auto Moderation Rule Create
+
+Sent when a rule is created. The inner payload is an [auto moderation rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object) object.
+
+#### Auto Moderation Rule Update
+
+Sent when a rule is updated. The inner payload is an [auto moderation rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object) object.
+
+#### Auto Moderation Rule Delete
+
+Sent when a rule is deleted. The inner payload is an [auto moderation rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object) object.
```

Comment:
All deletion events I've seen from the Gateway typically supply only an ID. Will this also be the case for auto moderation rule deletion?

Reply:
seems like there are some delete events like for channels and scheduled events that still include the full object for more context. We could keep the full object here to provide the most context around deletion unless there is a reason only the `id` is preferred